Code2Gether-Discord-Bot.Tests/ExcuseGeneratorTest.cs

```diff
@@ -60,18 +60,18 @@ public void Setup()
 
         [Test]
         public void EmbedAuthorHasValueTest() =>
-            Assert.IsTrue(_logic.Execute().Author.HasValue);
+            Assert.IsTrue(_logic.Execute().Result.Author.HasValue);
 
         [Test]
         public void EmbedColorHasValueTest() =>
-            Assert.IsTrue(_logic.Execute().Color.HasValue);
+            Assert.IsTrue(_logic.Execute().Result.Color.HasValue);
 
         [Test]
         public void EmbedHasTitleTest() =>
-            Assert.IsTrue(_logic.Execute().Title.Length > 0);
+            Assert.IsTrue(_logic.Execute().Result.Title.Length > 0);
 
         [Test]
         public void EmbedHasDescriptionTest() =>
-            Assert.IsTrue(_logic.Execute().Description.Length > 0);
+            Assert.IsTrue(_logic.Execute().Result.Description.Length > 0);
```

Comment:
I don't know how Async is usually handled in tests, but maybe you should consider using async-await here.

Reply:
Right. If I use `.Result()` iirc I only get Aggregate Exceptions